### PR TITLE
add music-wheel of color

### PR DESCRIPTION
**Pull request by manuscrypt**
Hi Josh

Here is our little contribution to the colluder. if you like it, we can build it out some day and maybe show a ring per octave or use it to compose more easily or the like. 

I'm Baaaaack!!! Elm has me again.

Ttys, Cheerio
Manuel

### CODE REVIEW

File: elm-package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (knewter) [inline elm-package.json]: Can you remove this file from the root?

**Review by knewter — CHANGES_REQUESTED** [review]
This is awesome!  Just checked it out.  Just looks like elm-package was run from the root and an artifact made it into the commit.  Pumped to merge it if you can remove that file.

File: elm-package.json (hunk 1 above)
Comment (manuscrypt) [reply elm-package.json]: i think, i just did, by pushing my changes?

Comment (manuscrypt) [reply elm-package.json]: or did I?

**Review by knewter — APPROVED** [review]

**Comment by knewter** [comment]
woot!  thanks, looks awesome

**Pull request merged**